CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased][unreleased]
 
+## [v1.10.0] - 2023-06-21
```

Comment:
looks like this changelog was not updated previously. 1.10.0 already exists (https://github.com/trycourier/courier-php/blob/main/src/CourierClient.php#L15). Mind updating this file as well as CourierClient.php to 1.11.0?